### PR TITLE
[Master] - Update the SEO descriptions of GraphQL BBEs 

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the SEO descriptions of GraphQL BBEs in the `master` branch.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/graphql-client-handle-partial-response/graphql_client_handle_partial_response.metatags [hunk 1]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL client written in Ballerina
+description: This example demonstrates handling partial data returned along with errors.
```
Comment (ThisaruGuruge) [inline examples/graphql-client-handle-partial-response/graphql_client_handle_partial_response.metatags:1]: Shouldn't we mention `GraphQL client` here?

File: examples/graphql-client-query-endpoint/graphql_client_query_endpoint.metatags [hunk 2]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL client written in Ballerina
+description: This example demonstrates connecting and interacting with a GraphQL server via a GraphQL client.
```
Comment (ThisaruGuruge) [inline examples/graphql-client-query-endpoint/graphql_client_query_endpoint.metatags:1]:
```suggestion
description: This example demonstrates connecting and interacting with a GraphQL server via the Ballerina GraphQL client.
```

File: examples/graphql-directives/graphql_directives.metatags [hunk 3]
```diff
@@ -1,2 +1,2 @@
-description: BBE on how to use graphql directives
+description: This example demonstrates using GraphQL directives.
```
Comment (ThisaruGuruge) [inline examples/graphql-directives/graphql_directives.metatags:1]:
```suggestion
description: This example demonstrates using GraphQL directives in Ballerina.
```

File: examples/graphql-field-interceptors/graphql_field_interceptors.metatags [hunk 4]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
-keywords: ballerina, ballerina by example, bbe, graphql, graphql service
+description: This example demonstrates executing custom logic by adding interceptors via GraphQL resolver functions.
```
Comment (ThisaruGuruge) [inline examples/graphql-field-interceptors/graphql_field_interceptors.metatags:1]:
```suggestion
description: This example demonstrates using interceptors on Ballerina GraphQL services to execute custom logic.
```

File: examples/graphql-file-upload/graphql_file_upload.metatags [hunk 5]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
+description: This example demonstrates uploading files to a GraphQL schema.
```
Comment (ThisaruGuruge) [inline examples/graphql-file-upload/graphql_file_upload.metatags:1]:
```suggestion
description: This example demonstrates implementing GraphQL service with file uploading capability.
```

File: examples/graphql-hierarchical-resource-paths/graphql_hierarchical_resource_paths.metatags [hunk 6]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
-keywords: ballerina, ballerina by example, bbe, graphql, graphql service
+description: This example demonstrates using hierarchical resource paths in the GraphQL resources.
```
Comment (ThisaruGuruge) [inline examples/graphql-hierarchical-resource-paths/graphql_hierarchical_resource_paths.metatags:1]:
```suggestion
description: This example demonstrates using hierarchical resource paths in GraphQL services.
```

File: examples/graphql-service-error-handling/graphql_service_error_handling.metatags [hunk 7]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
-keywords: ballerina, ballerina by example, bbe, graphql, error handling
+description: This example demonstrates handling errors that are returned by a GraphQL service.
```
Comment (ThisaruGuruge) [inline examples/graphql-service-error-handling/graphql_service_error_handling.metatags:1]:
```suggestion
description: This example demonstrates handling errors in a Ballerina GraphQL service.
```

File: examples/graphql-service-field-object/graphql_service_field_object.metatags [hunk 8]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina.
-keywords: ballerina, ballerina by example, bbe, graphql, graphql field
+description: This example demonstrates exposing information about a GraphQL field in a document.
```
Comment (ThisaruGuruge) [inline examples/graphql-service-field-object/graphql_service_field_object.metatags:1]:
```suggestion
description: This example demonstrates accessing information of a GraphQL field in a document.
```

File: examples/graphql-service-union-types/graphql_service_union_types.metatags [hunk 9]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
-keywords: ballerina, ballerina by example, bbe, graphql, graphql union type
+description: This example demonstrates defining union types as defined in the GraphQL specification.
```
Comment (ThisaruGuruge) [inline examples/graphql-service-union-types/graphql_service_union_types.metatags:1]:
```suggestion
description: This example demonstrates defining union types in a Ballerina GraphQL seervice.
```

File: examples/graphql-subscriptions/graphql_subscriptions.metatags [hunk 10]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
-keywords: ballerina, ballerina by example, bbe, graphql, subscription
+description: This example demonstrates defining GraphQL subscription operations.
```
Comment (ThisaruGuruge) [inline examples/graphql-subscriptions/graphql_subscriptions.metatags:1]:
```suggestion
description: This example demonstrates defining GraphQL subscription operation.
```

File: examples/graphql-mutations/graphql_mutations.metatags [hunk 11]
```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
-keywords: ballerina, ballerina by example, bbe, graphql, graphql mutation
+description: This example demonstrates defining GraphQL mutation operations.
```
Comment (ThisaruGuruge) [inline examples/graphql-mutations/graphql_mutations.metatags:1]:
```suggestion
description: This example demonstrates defining GraphQL mutation operation.
```

File: examples/graphql-client-handle-partial-response/graphql_client_handle_partial_response.metatags (hunk 1 above)
Comment (praneesha) [reply examples/graphql-client-handle-partial-response/graphql_client_handle_partial_response.metatags:1]: Updated.

File: examples/graphql-file-upload/graphql_file_upload.metatags (hunk 5 above)
Comment (praneesha) [reply examples/graphql-file-upload/graphql_file_upload.metatags:1]: Updated as: `This example demonstrates implementing a GraphQL service with the file uploading capability.`.

File: examples/graphql-service-field-object/graphql_service_field_object.metatags (hunk 8 above)
Comment (praneesha) [reply examples/graphql-service-field-object/graphql_service_field_object.metatags:1]: Updated.

File: examples/graphql-service-union-types/graphql_service_union_types.metatags (hunk 9 above)
Comment (praneesha) [reply examples/graphql-service-union-types/graphql_service_union_types.metatags:1]: Fixed the typo and updated.

File: examples/graphql-subscriptions/graphql_subscriptions.metatags (hunk 10 above)
Comment (praneesha) [reply examples/graphql-subscriptions/graphql_subscriptions.metatags:1]: Updated as: `This example demonstrates defining a GraphQL subscription operation.`.

File: examples/graphql-mutations/graphql_mutations.metatags (hunk 11 above)
Comment (praneesha) [reply examples/graphql-mutations/graphql_mutations.metatags:1]: Updated as: `This example demonstrates defining a GraphQL mutation operation.`.

File: examples/graphql-field-interceptors/graphql_field_interceptors.metatags (hunk 4 above)
Comment (praneesha) [reply examples/graphql-field-interceptors/graphql_field_interceptors.metatags]: Updated.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**